### ISSUE TITLE
Why not merge this back to the original?

### ISSUE THREAD

**Issue by boazpoolman** [BayBreezy/strapi-plugin-email-designer-5#10]
Great work migrating this plugin to Strapi 5!

I wast just wondering why you didn't merge this back to the original repository?
Seems rather confusing if there are multiple versions of the same plugin available.

**Comment by BayBreezy** [comment]
Hey @boazpoolman,

A few reasons:
- I am new to React. I would love for a React person to take a look and let me know if the code is ready for the world.
- The whole plugin structure of v4 vs v5 is vastly different.

I am open to this being merged into the main plugin. I would just need someone with a better understanding of how strapi works to help in doing so. I don't think I have deep enough knowledge to do that merge 🤣

**Comment by boazpoolman** [comment]
I might be able to help!

Just for my understanding, how did you start this new plugin?
I guess it was a fork of the original?

**Comment by BayBreezy** [comment]
Started it with the new plugin sdk.

I honestly would not have touched plugins if the new sdk wasn't developed.

**Comment by boazpoolman** [comment]
And then you copied the contents from the original plugin?

**Comment by BayBreezy** [comment]
Pretty much.. doing a few tweaks here and there

**Comment by begservice** [comment]
Just to be honest, the old plugin is not really under support here, merging that into the "OG" plugin would not help anyone. 
This is a very nice tool to make strapi even better than it is. @boazpoolman  is there a way to make it more of an strapi repository that somebody is moderating the merges into that? :)

**Comment by begservice** [comment]
i mean same as these plugin here: 
https://github.com/kucherenko/strapi-plugin-passwordless

Also fucking nice plugin, but really those plugins need to be more as features in Strapi would be awesome =)

**Comment by boazpoolman** [comment]
Yes, there are a lot of unmaintained plugins. That is just how Open Source works.
When one maintainer steps down, another one stands up. And that's great!

My point is that the original repository of a project holds an invaluable amount of information:
- It documents the decisions that were made in previous bugs and features.
- It documents bugs that some user might have had, but was never picked up yet.
- It lists all the features that we're suggested and brainstorms about those features.
- It holds a changelog of releases, but also a history of commits.
- It is the place that is referred to from the Strapi market, or other places on the web

It's amazing that people are willing to contribute to these plugins, so shout out to @BayBreezy for taking this task on!
But if you feel like a plugin is unmaintained, please contact the current maintainer/author and talk to them about it.
I'll bet most authors will be happy to make you a maintainer on the original repository to help them out. Or when they're looking to get out completely; move the to your name. That way we keep the Strapi community clean and neat.

**Comment by BayBreezy** [comment]
Yeah. I really wanted to use that passwordless plugin so bad in v4 but there so many issues that weren't getting any attention that i just left it alone. Found myself implementing the same passwordless logic manually :(.

I agree with your points @boazpoolman . Having one big repo where we can go and get all the info is great.
As for the original email designer, I have not seen the maintainer do anything on that repo in a minute. 

And like i said earlier in the convo, should a react noob (me) be in charge of a package that so many ppl are depending on? If this package blows up, it will force me to learn more React, which I am not a big fan of (Nuxt all the way 🚀). And I will have to reach out and bother the experts(@boazpoolman ) when i hit a road block.

When I put open source stuff out there, I always let ppl know that i am no expert LOLOL. 

That is why I said I am open to this being merged into the original repo. Maybe we should go over there and ask the maintainer what they think about this merge?

**Comment by boazpoolman** [comment]
You don't have to be responsible for the package in order to contribute to it.

I think opening an issue on the original repo asking about this topic is great!

Let's see what the author thinks.

**Comment by BayBreezy** [comment]
Done @boazpoolman

**Comment by BayBreezy** [comment]
Does anyone here know where I can see an example of Strapi's table pagination ? I want to add pagination to the custom email table but I am struggling to find a simple example. 

This way if you have hundreds of designs, you can page through them.

**Comment by ztxone** [comment]
> Does anyone here know where I can see an example of Strapi's table pagination ?

This is the docs of Strapi's Design System
https://design-system.strapi.io/?path=/docs/design-system-components-pagination--docs

**Comment by SalahAdDin** [comment]
@BayBreezy does it require to have an account in Underlying?
